acceptance/indy_acceptance/indy-tests/libraries/result.py

```diff
@@ -0,0 +1,127 @@
+"""
+Created on Nov 9, 2017
+
+@author: nhan.nguyen
+
+Containing classes to make the test result as a json.
+"""
+
+import json
+import time
+import os
+import errno
+from .constant import Colors
+
+
+class KeyWord:
```

Comment:
In Python there is no necessity to use classes as containers of static members as it would be in Java.
In Python a module is a natural container for static data and functions.

Reply:
Updated. Please refer to the latest code.